packages/dispatcher-converter/src/converter/amsDispatcherConfigConverter.js

```diff
@@ -24,6 +24,7 @@ const {
 const fs = require("fs");
 const glob = require("glob");
 const path = require("path");
+const { AVAILABLE_VHOSTS, AVAILABLE_FARMS } = require("../util/constants");
```

Comment:
We have already imported this class in Constants variable. We can directly use it from there for both the paths in symlinks functions.
Eg. Constants.AVAILABLE_VHOSTS 

Rest LGTM

Reply:
my bad , didn't realise ! just committed the changes. Thanks @prateek610  & @ManasMaji